samples/WinForms/Assets/Shaders/HelloTriangle.hlsl

```diff
@@ -0,0 +1,25 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
```

Comment:
This is identical to the existing shader from the more general samples, it should likely be linked rather than copied.

Reply:
ok

Reply:
done